package.json

```diff
@@ -6,6 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "server:build": "tsc --project server/tsconfig.json",
+    "server:start": "npm run server:build && node ./server/dist/index.js",
```

Comment:
add please one more script like "dev-mock-server": "npm-run-all --serial server:start dev" to use one terminal for running